include/graphblas/bsp1d/vector.hpp

```diff
@@ -2377,6 +2377,60 @@ namespace grb {
 #endif
 		}
 
+		/**
+		 * Creates a dense ALP/GraphBLAS vector. This constructor takes an initializer
+		 * list of values that will be copied into this vector.
```

Comment:
Maybe better to `@see` the base specification? The doxygen here will not make it to the user end-manual, and is only read by ALP developers themselves (i.e., the comments here could be considered internal, and typically should contain only notes that are relevant to ALP devs themselves)